File: src/python/t1431_KidsWithTheGreatestNumberOfCandies.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        result: list[bool] = []
        maxCand = max(candies)
        for i in range(len(candies)):
            result.append(candies[i] + extraCandies >= maxCand)
        return result

File: src/t1431_KidsWithTheGreatestNumberOfCandies.java
import java.util.Arrays;
import java.util.LinkedList;
import java.util.List;

public class t1431_KidsWithTheGreatestNumberOfCandies {
    public List<Boolean> kidsWithCandies(int[] candies, int extraCandies) {
        LinkedList<Boolean> result = new LinkedList<>();
        int max = Arrays.stream(candies).max().getAsInt();
        for(int i : candies)
            result.addLast(i+extraCandies >= max);
        return result;
    }
}
